core-skills/frontend-web-development/rhino-assessee.md

```diff
@@ -0,0 +1,42 @@
+# Rhino
+
+This assessment is composed of 3 discrete parts.
```

Comment:
Give some indication as to how long you need to set aside for this assessment.

Reply:
Addressed in 07558b6